src/FSAL/FSAL_CORTXFS/handle.c

```diff
@@ -1486,16 +1488,23 @@ static  inline fsal_status_t kvsfs_getattrs(struct fsal_obj_handle *obj_hdl,
 
 	myself =
 		container_of(obj_hdl, struct kvsfs_fsal_obj_handle, obj_handle);
-
-	retval = cfs_getattr(myself->cfs_fs, &cred,
-			     kvsfs_fh_to_ino(myself->handle), &stat);
+	/* TODO : Remove call to cfs_fh_from_ino
+	 * Currently, FH is temporarily obtained from the disk
+	 * because still FH is not in use at all the places.
+	 * When all such dependency is removed, the FH will be
+	 * extracted directly from kvsfs_fsal_obj_handle.
+	 */
+	//fh = myself->handle;
+	ino = kvsfs_fh_to_ino(myself->handle);
+	retval = cfs_fh_from_ino(myself->cfs_fs, ino, &fh);
```

Comment:
You need to sanity check retval here before extracting stat from FH.

Reply:
Done.